src/link_manager/LinkManagerStateMachineActiveStandby.cpp

```diff
@@ -476,12 +474,20 @@ void ActiveStandbyStateMachine::handleStateChange(LinkProberEvent &event, link_p
             mMuxPortPtr->postLinkProberMetricsEvent(link_manager::ActiveStandbyStateMachine::LinkProberMetrics::LinkProberActiveStart);
 
             mStandbyUnknownUpCount = 0;
+
+            if (mOscillationTimerAlive) {
+                cancelOscillationTimer();
+            }
```

Comment:
Not to add to this PR. Suggest move the check into cancellation helper function, and rename the function to tryCancel......()

Reply:
Addressed, thanks!